Incident_Process.md

```diff
@@ -0,0 +1,97 @@
+
+# The MacAdmins Slack Incident Process
```

Comment:
Smallest of nits, but we've moved pretty consistently elsewhere to separating: Mac Admins.

Reply:
Done